kube/services/aws-es-proxy/aws-es-proxy-deploy.yaml

```diff
@@ -22,6 +22,16 @@ spec:
         netvpc: "yes"
         GEN3_DATE_LABEL
     spec:
+      affinity:
+        nodeAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+          - weight: 100
+            preference:
+              matchExpressions:
+              - key: eks.amazonaws.com/capacityType
+                operator: In
+                values:
+                - ONDEMAND
```

Comment:
My only reason to set it as on demand was this and guppy potentially handle really long queries. It's more to prevent an issue coming up with long running queries being dropped and potentially hurting user experience. I figured it isn't as critical for guppy though because we normally run multiple instances of guppy, but only a single instance of es-proxy, so es-proxy going down has potential to interrupt the queries for all guppy pods, vs just 1 replica of guppy needing to rerun a query if a spot instance with guppy goes down 🤷‍♂️ .